### PR TITLE
change all places that use hard coded string to using constants

### PR DESCRIPTION
**Pull request by bill-mode**
- Added 2 constants maps, MODE_CONSTANTS, and CONSTANTS
    - MODE_CONSTANTS are constants used for Mode platform, it should not have any constants related to the app
    - CONSTANTS contains app-specific constants e.g. SENSOR_MODULE_KEY_PREFIX, EVENT_SENSOR_MODULE_LIST. These are constants we created for sensor cloud app, other apps might not use these constants.

### CODE REVIEW

File: sceval_frontend/src/controllers/ModeAPI.ts [hunk 1]
```diff
@@ -42,6 +40,30 @@ export interface ErrorResponse {
   status: number;
 }
 
+export class MODECONSTANTS {
```
Comment (reprimande) [inline sceval_frontend/src/controllers/ModeAPI.ts]: This is `class`. So I think `ModeConstants` is better for class name.

File: sceval_frontend/src/controllers/ModeAPI.ts [hunk 2]
```diff
@@ -42,6 +40,30 @@ export interface ErrorResponse {
...
+  public static MODE_API_BASE_URL: string = 'https://api.tinkermode.com/';
```
Comment (reprimande) [inline sceval_frontend/src/controllers/ModeAPI.ts]: It can be rewrited using only `static`. It's better using also `readonly` modifier.

Comment (reprimande) [reply sceval_frontend/src/controllers/ModeAPI.ts]: I think it's also use `module` and `export const` for this like as below.

```
module Constants {
    export const SOME_VALUE: string = "foo";
}
```

File: sceval_frontend/src/utils/Constants.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+
+export class CONSTANTS {
```
Comment (reprimande) [inline sceval_frontend/src/utils/Constants.ts]: Ditto

Comment (bill-mode) [reply sceval_frontend/src/utils/Constants.ts]: @reprimande, thanks for the feedback. I changed the class name and also changed it from class to namespace.

**Pull request merged**